src/utils.rs

```diff
@@ -177,8 +177,8 @@ pub(crate) fn install_tracing(is_verbose: bool) {
     let _ = create_dir_all(&log_dir);
 
     let file_appender = RollingFileAppender::builder()
-        .max_log_files(KEEP_LAST_N_DAYS)
-        .rotation(Rotation::DAILY)
+        .max_log_files(KEEP_LAST_N_FILE)
+        .rotation(Rotation::HOURLY)
```

Comment:
I believe you can use those log directories and it will remove most of logs:
```rust
    "trust_dns_proto=info",
    "parity-db=info",
    "libp2p_dns=info",
    "netlink_proto=info",
    "libp2p_tcp=info",
    "libp2p_ping=info",
    "libp2p_core=info",
    "libp2p_gossip=info",
    "multistream_select=info",
    "yamux=info",
    "libp2p_swarm=info",
    "trust_dns_resolver=info",
    "subspace_networking::node_runner=info",
    "libp2p_kad=info",
    "wasm-heap=info",
    "wasm_overrides=info",
    "wasmtime_cranelift=info",
    "rustls=info",
    "sync=info",
    "sub-libp2p=info",
    "txpool=info",
    "telemetry=info",
    "regalloc2=info",
    "frame_executive::final_checks=off",
    "frame_executive::execute_block=off",
```

Reply:
for now, our log output is nearly identical to monorepo outputs. Our default is `info`, so I don't think putting all these in `info` mode is necessary.